docs/usage/typescript.md

```diff
@@ -0,0 +1,53 @@
+# TypeScript
+
+## Regular TypeScript
+
+TypeScript code is executed via a Jupyter kernel. Markdown Exec will attempt to
+use any kernel that lists TypeScript as a supported language, however
+Markdown Exec is tested using the Jupyter kernel provided by the Deno project.
+See Deno's [installation instructions](https://docs.deno.com/runtime/getting_started/installation/)
+and [Jupyter kernel docs](https://docs.deno.com/runtime/reference/cli/jupyter/#quickstart)
+for more information.
+
+### Output capturing
+
+Outputs are captured just as they are in jupyter notebooks.
+
+````md exec="1" source="tabbed-left" tabs="Markdown|Rendered"
+```typescript exec="1"
+console.log("**Hello world!**");
+```
+````
+
+### Type checking
+
+TypeScript code blocks will be type checked using Deno's type checker.
+
+````md exec="1" source="tabbed-left" tabs="Markdown|Rendered"
+```typescript exec="yes" returncode="1"
+const x: string = 1;
+```
+````
+
+## TypeScript REPL console code
+
+Code blocks syntax-highlighted with the `tscon` identifier are also supported.
```

Comment:
question: Is `tscon` a thing? Like, does TypeScript have an interactive console/REPL too, like Python? Or did you just implement `tscon` because we have it for Python?